Recent change to sendgrid/rest is breaking sendgrid/sendgrid-go

#### Issue Summary

Recent change to sendgrid/rest is breaking sendgrid/sendgrid-go. Ran
`go get github.com/sendgrid/sendgrid-go`

getting errors when building:
`../../vendor/github.com/sendgrid/sendgrid-go/sendgrid.go:49:2: not enough arguments to return`

`../../vendor/github.com/sendgrid/sendgrid-go/sendgrid.go:49:22: DefaultClient.API undefined (type *rest.Client has no field or method API)`


#### Steps to Reproduce

1. `go get github.com/sendgrid/sendgrid-go`
2. `go build`
3. errors

rest.Client.API function does not exist anymore and is still being used by sendgrid/sendgrid-go.

#### Technical details:

* package: github.com/sendgrid/sendgrid-go version: v3.4.1
* go version go1.9 darwin/amd64

--- a/rest.go
+++ b/rest.go
@@ -124,6 +124,11 @@ func (c *Client) MakeRequest(req *http.Request) (*http.Response, error) {
 	return c.HTTPClient.Do(req)
 }
 
+// Function for support old implementation (deprecated)
+func (c *Client) API(request Request) (*Response, error) {
+	return Send(request)
+}
+
 // API is the main interface to the API.
 func (c *Client) Send(request Request) (*Response, error) {
 	// Build the HTTP request object.

--- a/rest.go
+++ b/rest.go
@@ -126,7 +126,7 @@ func (c *Client) MakeRequest(req *http.Request) (*http.Response, error) {
 
 // Function for support old implementation (deprecated)
 func (c *Client) API(request Request) (*Response, error) {
-	return Send(request)
+	return c.Send(request)
 }
 
 // API is the main interface to the API.

